decidim-admin/app/commands/decidim/admin/create_attachment.rb

```diff
@@ -28,7 +28,7 @@ def call
           @attachment.save!
           broadcast(:ok)
         else
-          if @attachment.errors.has_key? :file
+          if @attachment.errors.key? :file
```

Comment:
I see. I'll update the PR then to allow `has_` methods as proposed and change `keys?` usages to `has_keys?`.